test/samples/meson.build

```diff
@@ -0,0 +1,62 @@
+# Create core video, audio, and subtitle streams to construct files out of.
+video = custom_target('video.mkv',
+    output: 'video.mkv',
+    command: [ffmpeg, '-y', '-f', 'lavfi', '-i', 'testsrc=duration=2:size=1280x720', '@OUTPUT@'],
+)
+
+audio = custom_target('audio.flac',
+    output: 'audio.flac',
+    command: [ffmpeg, '-y', '-f', 'lavfi', '-i', 'sine=frequency=1000:duration=2', '@OUTPUT@'],
+)
+
+sub = join_paths(source_root, 'test', 'samples', 'sub.srt')
+
+samples = {
+    'eng_default.mkv':
+        [ffmpeg, '-y', '-i', video, '-i', audio, '-f', 'srt', '-i', sub,
```

Comment:
Please add `'-v', 'error'` or something similar to not spam logs with those commands.